src/ui/Histogram/UiHistogram.js

```diff
@@ -1,40 +1,16 @@
 /*
- * Copyright 2021 EPAM Systems, Inc. (https://www.epam.com/)
+ * Copyright 2022 EPAM Systems, Inc. (https://www.epam.com/)
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/**
- * @fileOverview UiHistogram
- * @author Epam
- * @version 1.0.0
- */
-
-// ********************************************************
-// Imports
-// ********************************************************
-
 import React from 'react';
 
-import TransfFunc from '../engine/TransFunc';
-
-// ********************************************************
-// Const
-// ********************************************************
+import TransfFunc from '../../engine/TransFunc';
 
 const DEFAULT_HEIGHT = 220;
 const NEED_TO_DRAW_VERTICAL_MARKS = false;
 
-// ********************************************************
-// Class
-// ********************************************************
-
-/**
- * Class UiHistogram some text later...
- */
 export default class UiHistogram extends React.Component {
```

Comment:
If it's possible please change this file to functional component :)

Reply:
Hi, so far without a functional component, since I checked if the files work separately, and left the old file for reference and verification.

Reply:
Okay